Class-Code/Game of Life/awu/sketch.js

```diff
@@ -1,14 +1,91 @@
-//Your Name(s)
-//Project Name
-//Date
+//Alex Wu
+//Game of Life
+// 3/15 2019
 
-/* A brief description of what the program is */
+/* A brief description of what the program is 
 
+This project intends to implement the elementary CA in a 2D situation
+and represent the dynacmic change of cells through animation. The cells
+randomized with 1 and 0(50|50% chance) in initial state. And the rules 
+follow the typical one
+
+*/
+let cell_console;
+let cell_size;
+let color = 250;
+let cell_length;
+let next_cell;
 
 function setup() {
     createCanvas(400, 400);
+    frameRate(5)
+    cell_length = 101;
+    // specify the length of column and row
+    cell_console = Array(cell_length);
+    for (i = 0; i < cell_length - 1; i++) {
+        cell_console[i] = Array(cell_length).fill(0);
+    }
+    //initialize columns and rows of cells
+    for (i = 1; i < cell_length - 1; i++) {
+        for (j = 1; j < cell_length - 1; j++) {
+            cell_console[i][j] = (random() < 0.5) ? 1 : 0;
+
+        }
+    }
+    cell_size = width / cell_length;
 }
 
 function draw() {
-    background(220);
+    background(color);
+    //draw the current cell
+    for (let i = 0; i < cell_length-1; i++) {
+        for (let j = 0; j < cell_length-1; j++) {
+            if (cell_console[i][j] == 1) {
+                noStroke();
+                fill(0);
+                rect(i * cell_size, j * cell_size, cell_size, cell_size);
+            }
+        }
+    }
+
+
+    //initialize the next cell
+    next_cell = Array(cell_length);
+    for (i = 0; i < cell_length - 1; i++) {
+        next_cell[i] = Array(cell_length).fill(0);
+    }
+
+
+    //compute the next cell
+    for (let a = 1; a < cell_length - 2; a++) {
+        for (let b = 1; b < cell_length - 2; b++) {
+            // given the cell at (a,b)
+            let neighbors = 0;
+            for (let m = -1; m < 2; m++) {
+                for (let f = -1; f < 2; f++) {
+                    neighbors += cell_console[a + m][b + f];
+                    //find the value of its neighbor
+                }
+            }
```

Comment:
This is perfectly functional, but perhaps a little obtuse. why `m` and `f`? Why `a` and `b`. You can reuse `i` and `j` since they are scoped appropriately with the `let` statement.